tests/test_strategies.py

```diff
@@ -80,21 +80,21 @@ def test_file(self):
         input = TEST_REPO_PATH / 'test_data/input_test_strategies/scenario_A.json'
         scenario.Scenario(load_json(input), input.parent)
 
-    # TEST SCENARIOS WITH BATTERY, FEEDIN AND EXTERNAL LOAD (Scenario A)
+    # TEST SCENARIOS WITH BATTERY, FEEDIN AND FIXED LOAD (Scenario A)
     def test_scenario_A(self):
         input = TEST_REPO_PATH / 'test_data/input_test_strategies/scenario_A.json'
         s = scenario.Scenario(load_json(input), input.parent)
         for strat in ['greedy', 'balanced', 'balanced_market', 'flex_window', 'peak_load_window']:
             s.run(strat, {})
 
-    # TEST with battery, feedin, extLoad and V2G (Scenario B)
+    # TEST with battery, feedin, fixedLoad and V2G (Scenario B)
```

Comment:
Here and in other comments (e.g. line 97): "feedin"?

Reply:
done